src/app/handlers/command.handler.ts

```diff
@@ -2,22 +2,22 @@ import { IMessage, IContainer, IHandler } from '../../common/types';
 import Constants from '../../common/constants';
 
 export class CommandHandler implements IHandler {
-  constructor(public container: IContainer) { }
+  constructor(public container: IContainer) {}
 
   public async execute(message: IMessage): Promise<void> {
-    const alias = this.build(message.content);
+    const command = this.build(message.content);
     const plugins = this.container.pluginService.plugins;
     const aliases = this.container.pluginService.aliases;
 
-    if (!alias) {
+    //checks to see if the user is actually talking to the bot
+    if (command === undefined) {
```

Comment:
Not a fan of explicit `=== undefined` better to just discretely use the truthiness of `command`.